packages/shared/src/components/cards/PostCard.tsx

```diff
@@ -68,30 +69,28 @@ export const PostCard = forwardRef(function PostCard(
     style,
     insaneMode,
     onReadArticleClick,
-    postCardVersion = 'v1',
     postModalByDefault,
     postEngagementNonClickable,
     ...props
   }: PostCardProps,
   ref: Ref<HTMLElement>,
 ): ReactElement {
   const onPostCardClick = () => onPostClick(post);
-  const isV1 = postCardVersion === 'v1';
-  const Containter = useMemo(
-    () =>
-      classed(
-        'div',
-        'relative flex flex-1',
-        postCardVersion === 'v1' ? 'flex-col' : 'flex-col-reverse',
-      ),
-    [postCardVersion],
-  );
+  const [isSharedPostShort, setSharedPostShort] = useState(true);
+  const containerRef = useRef<HTMLDivElement>();
+  const onSharedPostTextHeightChange = (height: number) => {
+    if (!containerRef.current) {
+      return;
+    }
+    setSharedPostShort(containerRef.current.offsetHeight - height < 40);
+  };
+  const isSharedPost = !!post.sharedPost;
```

Comment:
We have a type field for post. It should be used to indicate the card type instead of `sharedPost` object

Reply:
Used type instead 👍